### PR TITLE
Re-run and refactoring of some docs/tutorials + minor changes

### PR DESCRIPTION
**Pull request by marcocuturi**
- The notebook "Focus on Sinkhorn" now provides a guideline to solve the parallelization issue raised in #371. Essentially, when running `sinkhorn` on a single kernel matrix / for multiple pairs of histograms, one can achieve large speed ups (as mentioned a while time ago in [here](https://papers.nips.cc/paper_files/paper/2013/hash/af21d0c97db2e27e13572cbf59eb343d-Abstract.html), by parallelizing computations. For this vectorization to happen seamlessly with a simple `vmap`, one must pass explicitly a `min_iterations=max_iterations=...` to the Sinkhorn solver. This triggers a `scan` loop for sinkhorn iterations (instead of a fixed point iterator), and, as a result, greatly facilitates XLA optimization. This point was overlooked previously, and yields, as can be expected, enormous speed-ups in that case. Pinging @gjhuizing  who noticed it first. For now I am hoping this notebook highlights the problem enough for users, but we might consider making this trick more explicit.
- Monge gap notebook simplified, with some tools added to facilitate parameter tunings here and there. pinging @theouscidda6 
- `OTT` vs `POT` notebook corrected (was also mentioned in #371): this includes now comparisons with `POT` using a JAX backbone. Might be relevant for POT folks, pinging @rflamary @tommoral. Here the speedups obtained from jitting naturally arise from `n<8k`. When running higher point clouds, jitting doesn't improve kernel multiplication / lse applies much further.
- Clarification in `sinkhorn`: `n_iters` now counts **total** iterations (rather than numbers of `inner_iterations` loops, to avoid ambiguity), also added `n_iters` in `sinkhorn_divergence`.

### CODE REVIEW

**Comment by marcocuturi** [comment]
pushing without review because this is mostly docs and nothing really new on the codebase! Thanks!

File: docs/spelling/technical.txt [hunk 1]
```diff
@@ -36,6 +37,7 @@ binarized
 boolean
 centroids
 checkpointing
+chromatine
```
Comment (michalk8) [inline docs/spelling/technical.txt:40]: This is a typo, should be `chromatin`.

File: docs/spelling/technical.txt [hunk 2]
```diff
@@ -78,6 +81,7 @@ linearized
 logit
 macOS
 methylation
+mininum
```
Comment (michalk8) [inline docs/spelling/technical.txt:84]: This is also a typo.

**Comment by marcocuturi** [comment]
thanks!

**Pull request merged**